v0/destinations/freshmarketer/data/FRESHMARKETERIdentifyConfig.json

```diff
@@ -0,0 +1,194 @@
+[
+  {
+    "destKey": "emails",
```

Comment:
According to the doc, it is an array of hashes and `email` is of string type. Are they accepting string in emails?

Reply:
Yes, they accept `emails` in string type.